### PR TITLE
glance: update config template to newton

### PR DESCRIPTION
**Pull request by Itxaka**

### CODE REVIEW

**Comment by Itxaka** [comment]
[(5) Update templates for config files for Newton - Glance](https://trello.com/c/vygtc8vY/373-5-update-templates-for-config-files-for-newton-glance)

**Comment by Itxaka** [comment]
CI wont pass until the package is rebuilt with https://github.com/crowbar/crowbar-openstack/pull/657

File: chef/cookbooks/glance/templates/default/glance-api.conf.erb [hunk 1]
```diff
@@ -20,7 +20,6 @@ registry_client_insecure = <%= @keystone_settings["insecure"] %>
 debug = <%= node[:glance][:debug] ? "True" : "False" %>
 verbose = <%= node[:glance][:verbose] ? "True" : "False" %>
 log_file = <%= node[:glance][:api][:log_file] %>
-use_syslog = <%= node[:glance][:use_syslog] ? "True" : "False" %>
```
Comment (toabctl) [inline chef/cookbooks/glance/templates/default/glance-api.conf.erb]: AFAIK ```use_syslog``` is not deprecated.

Comment (Itxaka) [inline chef/cookbooks/glance/templates/default/glance-api.conf.erb]: http://docs.openstack.org/newton/config-reference/tables/conf-changes/glance.html#id3

Comment (toabctl) [inline chef/cookbooks/glance/templates/default/glance-api.conf.erb]: Then this doc is wrong. Doing a ```tox -egenconfig``` and then looking at the configs, ```use_syslog``` is still there. And it is the format that is deprecated, not ```use_syslog```. But it is confusing.

Comment (Itxaka) [inline chef/cookbooks/glance/templates/default/glance-api.conf.erb]: Interesting, I just cheked and indeed the option appears and its not deprecated (or at least it doesnt say anywhere in the comments that it is). Doc bugs?

Comment (Itxaka) [inline chef/cookbooks/glance/templates/default/glance-api.conf.erb]: reverted the use_syslog removal

**Comment by Itxaka** [comment]
@toabctl I reverted the use_syslog removal

**Pull request merged**